README.md

```diff
@@ -91,11 +91,35 @@ Superset provides:
 * Fast loading dashboards with configurable caching
 
 
-## Database Support
+## Supported Databases
 
 Superset speaks many SQL dialects through SQLAlchemy - a Python
-SQL toolkit that is compatible with most databases. A list of
-supported databases can be found
+SQL toolkit that is compatible with most databases. Here are some of the major database solutions that are supported:
+
+<p float="left">
+  <img src="https://i.ibb.co/qYyYdsr/redshift.png" alt="redshift" border="0" width="106" height="41" />
```

Comment:
are these urls ever going to break? Can we add the images to the repo under the apache license?

Reply:
It's a free imaging hosting site that appears good, so I used it to get this PR going, since I can't upload files to this repo.
If you can upload files to this repo and prefer that way, I can send you the images for you to upload, then I'll fix this PR with the relative links. Just let me know what you prefer! :)

Reply:
Never heard of `https://i.ibb.co`, wouldn't rely on it.

Your images can be part of this PR (if you can PR you can add images).